src/lib/server.js

```diff
@@ -23,25 +38,35 @@ module.exports.init = function (io) {
         });
 
         socket.on('client:tick', (data) => {
-            const currentUnixTime = +(new Date());
-            gameState[socket.id] = {
+            const currentUnixTime = utils.timestamp();
+            Object.assign(gameState[socket.id], {
                 time: currentUnixTime,
                 facing: data.facing,
                 moving: data.moving,
                 velocity: data.velocity,
                 coordinates: data.coordinates
-            };
+            });
+
+            determineHits(gameState, io);
```

Comment:
It's a little dirty pulling `io` through here, perhaps pull broadcasting out into a function?